src/azul/plugins/metadata/hca/transform.py

```diff
@@ -773,6 +775,21 @@ def _is_dcp2_matrix(self, file: api.File) -> bool:
         else:
             return False
 
+    def file_source(self, submitter_id: str) -> Optional[str]:
+        """
+        Return the human-readable version of the name that was used to generate
+        the submitter UUID.
+        """
+        try:
```

Comment:
The second try should be nested  and fall-throughs should be avoided if possible.

Reply:
The fact that this is essentially the same code twice bugged me, too.